ui/src/ReactCustomHooks/useWebSocket/useWebSocket.hook.js

```diff
@@ -1,5 +1,5 @@
 import { useState, useEffect } from 'react';
-import { NO_OP, EMPTY_OBJ } from 'Utils';
+import { NO_OP, EMPTY_OBJ } from '../../Utils/index.js';
```

Comment:
Good spot/fix - wonder if we can share our webpack build aliases with the storybook build as well? (which would allow us to use `Utils` as it was before)

Reply:
Good idea - I've updated the storybook webpack.config.js file with the build aliases. All seems happy, so I've reverted this change.